content/en/blog/_posts/2020-11-02-remembering-dan-kohn.md

```diff
@@ -0,0 +1,16 @@
+---
+layout: blog
+title: "Remembering Dan Kohn"
+date: 2020-11-02
+slug: remembering-dan-kohn
+---
+
+**Author**: The Kubernetes Steering Committee
+
+Dan Kohn was instrumental in getting Kubernetes and CNCF community to where it is today. He shared our values, motivations, enthusiasm, community spirit and helped the Kubernetes community to become the best that it could be. He loved getting people together to solve problems big and small. He enabled people and has served to launch many a career of folks in the community.
+
+Dan built a coalition around the nascent Kubernetes project and turned that into a cornerstone to build the larger cloud native space. He loved challenges especially ones where the payoff was great like building world wide communities, spreading the love of open source, and helping diverse, underprivileged communities and students to get a head start in Technology.
```

Comment:
nit to add a comma to improve readability: 

He loved challenges, especially ones where the payoff was great like ...